kwc-tracking.html

```diff
@@ -467,18 +474,23 @@
                     this._tokenChanged = _tokenChanged.bind(this);
                     this._trackUserError = _trackUserError.bind(this);
                     this._initializeTracking = _initializeTracking.bind(this);
+                    this._trackSystemError = _trackSystemError.bind(this);
+                    this._clearQueueDispatch = _clearQueueDispatch.bind(this);
                     window.addEventListener('tracking-event', this._trackBasicEvent);
                     window.addEventListener('page-view', this._trackPageView);
                     window.addEventListener('token-changed', this._tokenChanged);
                     window.addEventListener('error-event', this._trackUserError);
                     window.addEventListener('initialize-tracking', this._initializeTracking);
+                    window.addEventListener('error', this._trackSystemError);
+                    window.addEventListener('beforeunload', this._clearQueueDispatch);
```

Comment:
I think we should remove this listener.. Maybe the callback could be an 'onWindowUnload' or whatever and that could remove the listener..

Reply:
Good point, thanks @vikdoro. I'll update to ensure that this is always removed.